packages/vite/src/client/client.ts

```diff
@@ -84,9 +84,9 @@ async function handleMessage(payload: HMRPayload) {
             ) as HTMLLinkElement[]
           ).find((e) => e.href.includes(path))
           if (el) {
-            const newPath = `${path}${
-              path.includes('?') ? '&' : '?'
-            }t=${timestamp}`
+            const newPath =
+              base +
+              `${path.slice(1)}${path.includes('?') ? '&' : '?'}t=${timestamp}`
```

Comment:
```suggestion
              `${base}${path.slice(1)}${path.includes('?') ? '&' : '?'}t=${timestamp}`
```

Reply:
Could you also please run the formatter after this change? Could change the suggestion

Reply:
Thanks! Should I also update [this `base` concatenation](https://github.com/ElMassimo/vite/blob/fix-stylesheet-hmr-base/packages/vite/src/client/client.ts#L332-L334) for consistency?